app/Http/Controllers/Api/UsersController.php

```diff
@@ -288,4 +290,19 @@ public function assets($id)
         $assets = Asset::where('assigned_to', '=', $id)->with('model')->get();
         return (new AssetsTransformer)->transformAssets($assets, $assets->count());
     }
+
+    /**
+     * Return JSON containing a list of license seats assigned to a user.
+     *
+     * @author [O. Loesch] [<github@loesch.io]
+     * @since [v4.11]
+     * @param $userId
+     * @return string JSON
+     */
+    public function license_seats($id)
+    {
+        $this->authorize('view', User::class);
```

Comment:
Shouldn't we be checking that the accessor can see this particular user, not just users in general?

Reply:
Ah, it's like this in `assets()`.  Probably older code (`@since [v3.0]`)?